extensions/integration-tests/src/notebook.util.ts

```diff
@@ -73,7 +75,9 @@ export const pythonKernelMetadata = {
 
 export function writeNotebookToFile(pythonNotebook: azdata.nb.INotebookContents): vscode.Uri {
 	let notebookContentString = JSON.stringify(pythonNotebook);
-	let localFile = tempWrite.sync(notebookContentString, 'notebook.ipynb');
+	let localFile = path.join(os.tmpdir(), 'notebook' + Math.floor(Math.random() * 101) + '.ipynb');
```

Comment:
Does this actually guarantee that no other file will exist? E.g. is the tmpdir unique? Just wondering if you need to handle conflicts at all.